netutils/ip.py

```diff
@@ -157,6 +157,30 @@ def ip_subtract(ip: str, val: int) -> str:
     """
     return str(ipaddress.ip_address(ip) - val)
 
+def is_classfull(ip_network: str) -> bool:
+    """Determines if a CIDR network address is within unicast class full boundaries.
+    
+    The following class boundaries are checked:
+    0.0.0.0/8 -> 127.0.0.0/8 as Class A
+    128.0.0.0/16 -> 191.255.0.0/16 as Class B
+    192.0.0.0/24 -> 223.255.255.0/24 as Class C
+
+    Example:
```

Comment:
Can you see how this renders in mkdocs? I think you may need the plural form

Reply:
Fixed in latest commit.